mitodl/mit-learn
# Merge `main` into `nextjs`

### What are the relevant tickets?

Relates to https://github.com/mitodl/hq/issues/5395

### Description (What does it do?)


Updates the Next.js branch with changes in `main`.


### How can this be tested?


Ensure that the branch runs locally and everything looks in order and recent changes in `main` have been applied, with attention to these PRs:

- https://github.com/mitodl/mit-learn/pull/1681
- https://github.com/mitodl/mit-learn/pull/1689
- https://github.com/mitodl/mit-learn/pull/1698
- https://github.com/mitodl/mit-learn/pull/1695
- https://github.com/mitodl/mit-learn/pull/1699
